api/MediaElementAudioSourceNode.json

```diff
@@ -129,10 +129,10 @@
               "version_added": true
             },
             "safari": {
-              "version_added": true
+              "version_added": false
```

Comment:
I'm fairly sure this is wrong. https://mdn-bcd-collector.appspot.com/tests/api/MediaElementAudioSourceNode in Safari 14 shows an exception being thrown in a way that only makes sense if the property is supported, and the result of the test is null. Is the update code treated null as false? Is so, changes like this one are very likely. It would be great to handle such cases separately.

One can also tell from https://github.com/WebKit/webkit/blob/master/Source/WebCore/Modules/webaudio/MediaElementAudioSourceNode.idl that `mediaElement` has been in that IDL file for a very long time.

Reply:
Hmm, some version must have returned `false` in the results, as we've made sure that `null` is just that, and an all `null` feature will not be set as `false`.  I'll look into this some more; for now I set it manually by going to https://mdn-bcd-collector.appspot.com/tests/api/MediaElementAudioSourceNode/mediaElement in Safari 6.